bpf/bpf_host.c

```diff
@@ -36,7 +46,7 @@
 #include "lib/nodeport.h"
 #include "lib/eps.h"
 
-#if defined(FROM_HOST) && (defined(ENABLE_IPV4) || defined(ENABLE_IPV6))
+#if defined(ENABLE_IPV4) || defined(ENABLE_IPV6)
```

Comment:
I can't remove it for now because [we compile test with both IPv4 and IPv6 disabled](https://github.com/cilium/cilium/blob/d613dea5d52494da703e72ef0011e2849f7f19ce/bpf/Makefile#L37). I'm wondering if that makes sense though. If we decide it doesn't, I can remove it in a follow up.